test/suites/standard2/telemetry.robot

```diff
@@ -13,7 +14,8 @@ Test Tags           restart    slow
 
 
 *** Variables ***
```

Comment:
I tlooks like ${JOURNAL_CURSOR} var declared in this block is never used. Can you please check it?

Reply:
Done